NitroxClient/GameLogic/SimulationOwnership.cs

```diff
@@ -53,6 +77,10 @@ public void ReceivedSimulationLockResponse(NitroxId id, bool lockAquired, Simula
             {
                 SimulateEntity(id, lockType);
             }
+            else if (!OtherPlayerHasAnyLock(id))
+            {
+                AddSimulationLockForUnknown(id, lockType);
+            }
```

Comment:
If this happens, then it looks like a result of a race condition; i.e., lock acquired before us but broadcasted later.  Should we just ignore this case and allow future lock packets to update the list?